vhost-device-scmi/src/main.rs

```diff
@@ -68,18 +69,17 @@ struct ScmiArgs {
 }
 
 pub struct VuScmiConfig {
-    socket_path: String,
+    socket_path: PathBuf,
     devices: DeviceDescription,
 }
 
 impl TryFrom<ScmiArgs> for VuScmiConfig {
     type Error = String;
 
     fn try_from(cmd_args: ScmiArgs) -> Result<Self> {
-        if cmd_args.socket_path.is_none() {
+        let Some(socket_path) = cmd_args.socket_path else {
             return Result::Err("Required argument socket-path was not provided".to_string());
```

Comment:
hm... probably it should simply not be an Option then :P.

nothing to change for this PR of course...

Reply:
Done! #572 :p

Reply:
The corresponding command line option itself is optional but no longer optional if this place is reached.